README.md

```diff
@@ -123,3 +123,8 @@ yarn lint
 - Create new PR with rewarder/\*\* branch name
 - Create newRewarderConfig.json file, fill out value keys from newRewarderConfig.sample.json
 - Once proposed gnosis safe signatures are signed and contracts are verified, we will merge your PR!
+
+## Update Pool allocation
+
+- Create new PR with allocation/\*\* branch name
```

Comment:
Can you write or link to an example PR?  Maybe create a PR then close it and link it?

Reply:
Will do after gnosis safe queue is empty, good idea :))

Reply:
https://github.com/trisolaris-labs/trisolaris_core/pull/95